test/unit/refresh.js

```diff
@@ -462,7 +462,6 @@ describe('Unit tests for swiftserver:refresh', function () {
         })
 
         it('contains no Health references', function () {
-          assert.noFileContent('Package.swift', 'Health')
```

Comment:
Health needs to be an optional module, since the new "blank" application type doesn't depend on `Health`. This assert should remain.